.github/workflows/release.yml

```diff
@@ -0,0 +1,22 @@
+name: Release
+
+on:
+  release:
+    types: [published]
+
+jobs:
+
+  publish-release:
+    runs-on: ubuntu-latest
+    environment: publish
+    steps:
+      - uses: actions/checkout@v2
+      - name: Build new Docker image
+        run: "bash starter_scripts/build_docker_runner_image.sh"
+      - name: Docker login
+        run: echo "$SECRET_DOCKER_TOKEN" | docker login --username "$SECRET_DOCKER_USER_NAME" --password-stdin
+        env: # Set the secret as an input
+          SECRET_DOCKER_USER_NAME: ${{ secrets.DOCKER_USER_NAME }}
+          SECRET_DOCKER_TOKEN: ${{ secrets.DOCKER_TOKEN }}
+      - name: Push new Docker images
+        run: "bash starter_scripts/push_docker_runner_image.sh latest ${{ github.event.release.tag_name }}"
```

Comment:
Quotes for ${{ github.event.release.tag_name }}

Reply:
The whole string is already encapsulated by quotes....